packages/golang/packaging

```diff
@@ -0,0 +1,4 @@
+set -e
+
+tar xzf golang/go1.6.2.linux-amd64.tar.gz
```

Comment:
There is a good practice to add a comment with a link to original page, where that package was taken.